src/components/Sidebar/components/NavItem/NavItem.js

```diff
@@ -73,20 +94,26 @@ const NavItem = ({
   defaultIcon,
   selectedIcon,
   selected,
+  disabled,
   onClick
-}) => (
-  <ListItem
-    selected={selected}
-    secondary={secondary}
-    visible={visible}
-    onClick={onClick}
-  >
-    {defaultIcon && selectedIcon && selected ? selectedIcon : defaultIcon}
-    <NavLabel secondary={secondary} visible={visible}>
-      {label}
-    </NavLabel>
-  </ListItem>
-);
+}) => {
+  const icon = getIcon({ defaultIcon, selected, selectedIcon, disabled });
+
+  return (
+    <ListItem
```

Comment:
I'd like to make the Sidebar keyboard navigable (currently it's not in the tab order).

I've tried adding `role="button" tabindex="0"` here, but this doesn't work (the tabindex is removed on build).

@fernandofleury do you know a better way?

Reply:
React uses `tabIndex` (note the uppercase `I`). `<a>` with an `href` attribute should be keyboard focusable by default. 

`role="button"` doesn't seem appropriate here, since it changes the semantics (buttons are for interactions, anchors for navigation).

Reply:
So do you think we should have a `role="link"` ? It's not really a link in the href sense either, as this is routing. I'm not entirely sure what best practices are here @connor-baer

Reply:
What's the underlying element that's being rendered? What happens when an item is clicked?

Reply:
➡️ v2

This PR won't fix this to avoid hacky solutions or breaking changes. We'll wrap the NavItems in the apps in anchor tags to make the Sidebar keyboard navigable.